src/nm-core-utils.h

```diff
@@ -283,7 +283,10 @@ gboolean nm_utils_file_set_contents (const gchar *filename,
 char *nm_utils_machine_id_read (void);
 gboolean nm_utils_machine_id_parse (const char *id_str, /*uuid_t*/ guchar *out_uuid);
 
-guint8 *nm_utils_secret_key_read (gsize *out_key_len, GError **error);
+gboolean nm_utils_secret_key_read (guint8 **out_secret_key,
```

Comment:
Now that we have "nm_utils_secret_key_get", do we still need to expose in the cure header file "nm_utils_secret_key_read" or can we just make it a static function?